isisd/isis_cli.c

```diff
@@ -1436,6 +1436,50 @@ void cli_show_isis_srgb(struct vty *vty, struct lyd_node *dnode,
 		yang_dnode_get_string(dnode, "./upper-bound"));
 }
 
+/*
+ * XPath: /frr-isisd:isis/instance/segment-routing/srlb
+ */
+DEFPY (isis_sr_local_block_label_range,
+       isis_sr_local_block_label_range_cmd,
+       "segment-routing local-block (16-1048575)$lower_bound (16-1048575)$upper_bound",
+       SR_STR
+       "Segment Routing Local Block label range\n"
+       "The lower bound of SRLB (16-1048575)\n"
+       "The upper bound of SRLB (16-1048575)\n")
+{
+	nb_cli_enqueue_change(vty, "./segment-routing/srlb/lower-bound",
+			      NB_OP_MODIFY, lower_bound_str);
+	nb_cli_enqueue_change(vty, "./segment-routing/srlb/upper-bound",
```

Comment:
I don't see a check for this, so I thought I'd ask... What happens if the user accidentally inputs a negative range (where the lower bound is higher than the higher bound)? Might be good to check for this and spit an error rather than risk a problem later in the code (?).

Reply:
Check are done through pre-validate functions for SRLB and SRGB to check that ranges are not overlap each other. Look at isis_instance_segment_routing_srgb_pre_validate() and isis_instance_segment_routing_srlb_pre_validate() functions in isis_nb_config.c file.

Reply:
It is not possible to enter a negative value through CLI due to the `[(16-1048575) (16-1048575)]` pattern matching. Value must be within the interval 16 - 1048575.

You could also have a look to example I post to answer to @manuhalo comments above.